plaso/parsers/czip_plugins/oxml.py

```diff
@@ -72,13 +69,15 @@ def __init__(self):
     self.template = None
     self.total_time = None
 
-
-class OpenXMLParser(interface.FileObjectParser):
+class OpenXMLPlugin(interface.CompoundZIPPlugin):
   """Parse metadata from OXML files."""
 
-  NAME = 'openxml'
+  NAME = 'czip_oxml'
```

Comment:
Here I repeated "czip_" because other plugins do the same, but since others don't, I'd rather change it to "oxml" or "openxml", so we can use `--parser czip/oxml` instead. Let me know which one do you prefer.

Reply:
Good point, I've changed to just oxml